packages/actor-query-operation-construct/lib/ActorQueryOperationConstruct.ts

```diff
@@ -10,6 +10,7 @@ import type * as RDF from '@rdfjs/types';
 import type { AsyncIterator } from 'asynciterator';
 import { getTerms, getVariables, uniqTerms } from 'rdf-terms';
 import { Algebra } from 'sparqlalgebrajs';
+import { KeysQueryOperation } from './../../context-entries';
```

Comment:
This should refer to the actual package name, instead of being a relative path.

Reply:
For some reason the linter ask me to 

```
 error  '@comunica/context-entries' should be listed in the project's dependencies. Run 'npm i -S @comunica/context-entries' to add it  import/no-extraneous-dependencies
```

when I do `import { KeysQueryOperation } from '@comunica/context-entries';` and installing the dependency does't work
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
```

Reply:
>> when I do import { KeysQueryOperation } from '@comunica/context-entries';

This is correct.

You'll have to add the dependency manually to the package.json of this package, and then run a yarn install on the comunica root.